tests/unit/helpers/number-format-test.js

```diff
@@ -0,0 +1,17 @@
+import { numberFormat } from 'dummy/helpers/number-format';
+import { module, test, skip } from 'qunit';
+
+module('Unit | Helper | number format');
+
+// Works in Chrome but doesn't work in terminal
```

Comment:
@abought, the linked ticket it is showing the exact thing that i'm seeing.  If I do toLocaleString(1234) in phantomJS, it just returns 1234.  In the browser, it would be 1,234.